invenio_cli/cli.py

```diff
@@ -156,6 +156,16 @@ def destroy(verbose):
     commands.destroy()
 
 
+@cli.command()
+@click.option('--verbose', default=False, is_flag=True, required=False,
```

Comment:
moderate: This option is not used anywhere right? I think is a legacy from when we tried logging the output, might not be needed for now (I think is better to remove).